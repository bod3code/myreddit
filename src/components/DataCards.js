import React from 'react';
import Card from '@material-ui/core/Card';
import CardActions from '@material-ui/core/CardActions';
import CardContent from '@material-ui/core/CardContent';
import Button from '@material-ui/core/Button';
import Typography from '@material-ui/core/Typography';
import { makeStyles } from '@material-ui/core/styles';
const useStyles = makeStyles(theme => ({
  card: {
      margin: theme.spacing(1,1,1,1)
  },
  title: {
      fontSize: 14,
  }
}));

const DataCards = (props) => {
  const classes = useStyles();

    const data = props.data;
    function formatDate(date) {
      var d = new Date(date),
          month = '' + (d.getMonth() + 1),
          day = '' + d.getDate(),
          year = d.getFullYear();
  
      if (month.length < 2) month = '0' + month;
      if (day.length < 2) day = '0' + day;
  
      return [year, month, day].join('-');
    };

    return(
    data.hits.map(item => (
    <Card key={item.url} className={classes.card} raised={true}>
  <CardContent>
    <Typography className={classes.title} color="textSecondary" gutterBottom>
    {item.author}
    </Typography>
    <Typography variant="h5" component="h2">
    {item.title}
    </Typography>
    <Typography color="textSecondary">
    {formatDate(item.created_at)}
    </Typography>
    <Typography variant="body2" component="p">
    </Typography>
  </CardContent>
  <CardActions>
    <Button size="small" href={item.url}>Learn More</Button>
  </CardActions>
</Card>
    )
 ))};

 export default DataCards;