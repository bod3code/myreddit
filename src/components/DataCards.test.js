import React from 'react';
import DataCards from './DataCards';
import renderer from 'react-test-renderer';
const json = require('./defaultResponse.json');
test('cards change when data changes', () => {
  const component = renderer.create(
    <DataCards data={json}></DataCards>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
  });