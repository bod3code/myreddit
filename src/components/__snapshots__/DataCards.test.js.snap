// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cards change when data changes 1`] = `
Array [
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        dwwoelfel
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        Relicensing React, Jest, Flow, and Immutable.js
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2017-09-22
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="https://code.facebook.com/posts/300798627056246"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        y4m4b4
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        Explaining React's license
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2017-08-18
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="https://code.facebook.com/posts/112130496157735/explaining-react-s-license/"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        markthethomas
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        React 16
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2017-09-26
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="https://facebook.github.io/react/blog/2017/09/26/react-v16.0.html"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        apetresc
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        Facebook Announces React Fiber, a Rewrite of Its React Framework
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2017-04-18
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="https://techcrunch.com/2017/04/18/facebook-announces-react-fiber-a-rewrite-of-its-react-framework/"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        pouwerkerk
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        Painting with Code: Introducing our new open source library React Sketch.app
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2017-04-25
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="http://airbnb.design/painting-with-code/"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        vjeux
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        Create React Apps with No Configuration
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2016-07-22
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="https://facebook.github.io/react/blog/2016/07/22/create-apps-with-no-configuration.html"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        WillAbides
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        Add Reactions to Pull Requests, Issues, and Comments
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2016-03-10
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        fanf2
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        Vue.js vs. React
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2017-08-19
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="https://vuejs.org/v2/guide/comparison.html#React"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        clessg
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        React v15.0
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2016-04-07
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="https://facebook.github.io/react/blog/2016/04/07/react-v15.html"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        tm33
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        Free React.js Fundamentals Course
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2016-03-01
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="http://courses.reactjsprogram.com/courses/reactjsfundamentals"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        uzername
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        Svelte 3: Rethinking Reactivity
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2019-04-22
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="https://svelte.dev/blog/svelte-3-rethinking-reactivity"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        BafS
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        React Native for OS X
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2016-03-29
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="https://github.com/ptmt/react-native-desktop"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        kawera
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        Simple React Patterns
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2017-11-09
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="http://lucasmreis.github.io/blog/simple-react-patterns/"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        bratao
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        ReactOS releases 0.4.8 with experimental Vista/7/10 software compatibility
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2018-04-15
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="https://reactos.org/project-news/reactos-048-released"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        ika
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        Blueprint – A React UI toolkit for the web
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2016-11-12
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="http://blueprintjs.com/"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-elevation8 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      <p
        className="MuiTypography-root makeStyles-title-2 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
      >
        ggregoire
      </p>
      <h2
        className="MuiTypography-root MuiTypography-h5"
      >
        Evergreen: a React UI Framework built by Segment
      </h2>
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        2018-10-16
      </p>
      <p
        className="MuiTypography-root MuiTypography-body2"
      />
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
        href="https://evergreen.segment.com"
        onBlur={[Function]}
        onDragEnd={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Learn More
        </span>
      </a>
    </div>
  </div>,
]
`;
